Dockerfile

```diff
@@ -5,10 +5,15 @@ ADD requirements.txt /env/requirements.txt
 RUN pip install -r /env/requirements.txt
 
 # Add application to container.
-ADD beluga /app/beluga/
+ADD . /app/
 
 WORKDIR /app/
 
-ENTRYPOINT [ "python3" ]
-
-CMD [ "beluga/app.py" ]
+# Hot reload on app from code modifications.
+CMD gunicorn \
+    --reload \
+    --bind 0.0.0.0:80 \
+    --workers 4 \
```

Comment:
I think these should vary based on env, maybe implementable with a simple bash script. In particular, I think we should turn off reloading in production, have one (maybe two, depending on concurrency properties of the program) worker in dev, and bind to 127.0.0.1 in dev (for security, so only we can access our dev server by default)

Reply:
Agreed here. I like the bash script entrypoint approach.